cmd/ucpd/ucp-self-hosted-dev.yaml

```diff
@@ -41,7 +41,7 @@ planes:
         Applications.Core: "http://localhost:8080"
         Applications.Messaging: "http://localhost:8081"
         Applications.Dapr: "http://localhost:8081"
-        Applications.Datastores: "http://localhost:8081"
+        Applications.Datastores: "http://localhost:8080"
```

Comment:
What is the reason this has changed to 8080?

Reply:
We're transitioning all resource providers to be serviced by one service 'radius' (as opposed to two currently). Hence the updates to port numbers.